bentoml/_internal/frameworks/transformers.py

```diff
@@ -106,427 +65,209 @@ def check_flax_supported() -> None:  # pragma: no cover
-            user-defined additional python objects to be saved alongside the model,
-            e.g. a tokenizer instance, preprocessor function, model configuration json
         metadata (:code:`Dict[str, Any]`, `optional`,  default to :code:`None`):
             Custom metadata for given model.
-        model_store (:mod:`~bentoml._internal.models.store.ModelStore`, default to :mod:`BentoMLContainer.model_store`):
-            BentoML modelstore, provided by DI Container.
+
     Returns:
-        :obj:`~bentoml.Tag`: A :obj:`tag` with a format `name:version` where `name` is the user-defined model's name, and a generated `version` by BentoML.
+        :obj:`~bentoml.Tag`: A :obj:`tag` with a format `name:version` where `name` is
+        the user-defined model's name, and a generated `version`.
 
     Examples:
 
     .. code-block:: python
 
-        from transformers import AutoModelForQuestionAnswering, AutoTokenizer
         import bentoml
-        model = AutoModelForQuestionAnswering.from_pretrained("gpt2", from_flax=True)
-        tokenizer = AutoTokenizer.from_pretrained("gpt2", from_flax=True)
 
-        # transfer training and modification goes here
-        ...
+        from transformers import pipeline
 
-        tag = bentoml.transformers.save("flax_gpt2", model=model, tokenizer=tokenizer)
-    """
-    if not check_model_type(obj):
-        if not LazyType["ext.TransformersPipeline"](
-            "transformers.pipelines.base.Pipeline"
-        ).isinstance(obj):
-            raise BentoMLException(
-                "`model` is neither a Transformers pipeline nor a Transformers model."
-            )
+        generator = pipeline(task="text-generation")
+        tag = bentoml.transformers.save_model("text-generation-pipeline", generator)
+
+        # load the model back:
+        loaded = bentoml.transformers.load_model("text-generation-pipeline:latest")
+        # or:
+        loaded = bentoml.transformers.load_model(tag)
+    """  # noqa
+    if not LazyType["ext.TransformersPipeline"](
+        "transformers.pipelines.base.Pipeline"
+    ).isinstance(pipeline):
+        raise BentoMLException(
+            "`pipeline` must be an instance of `transformers.pipelines.base.Pipeline`"
+        )
 
-    check_flax_supported()  # pragma: no cover
-    context: t.Dict[str, t.Any] = {
-        "framework_name": "transformers",
-        "pip_dependencies": [f"transformers=={get_pkg_version('transformers')}"],
-        "pipeline": False,
-        "task": False,
-    }
-    options: t.Dict[str, t.Any] = {
-        "model": "",
-        "tokenizer": False,
-        "feature_extractor": False,
-    }
+    context = ModelContext(
+        framework_name="transformers",
+        framework_versions={"transformers": get_pkg_version("transformers")},
+    )
+    options = TransformersOptions(task=pipeline.task)
+
+    if signatures is None:
+        signatures = {
+            "__call__": {"batchable": True},
+        }
+        logger.info(
```

Comment:
lets make this a debug log instead

Reply:
I think this should be info level, it should show up by default.

Reply:
I also think `info` is maybe the right logging level. Please let me know if you feel strong about this.

Reply:
I don't have a strong opinion on this. we can just keep it as it 👍🏼